fern/versions/v1/pages/integrating-dafpay/transactions.mdx

```diff
@@ -35,7 +35,7 @@ Once a nonprofit receives the funds from the DAF provider and identifies the spe
     
 If a grant status is updated in your fundraising platform’s dashboard, make sure to use the [Update Grant](/api/grants/update) API so we can reflect those changes on our end.
 
-If a grant status is updated directly on the Chariot dashboard, subscribe to the `grant.updated` [Webhook Event](https://chariot-preview-43a7ae5e-30ae-432a-913c-d6d9ee4459c5.docs.buildwithfern.com/api/webhooks) to receive notifications.
+If a grant status is updated directly on the Chariot dashboard, subscribe to the `grant.updated` [Webhook Event](https://docs.givechariot.com/api/webhooks) to receive notifications.
```

Comment:
these should always be done as relative links to ensure preview versions of the doc can work seamlessly